langchain/src/memory/zep.ts

```diff
@@ -67,8 +67,7 @@ export class ZepMemory extends BaseChatMemory implements ZepMemoryInput {
     } catch (error) {
       // eslint-disable-next-line no-instanceof/no-instanceof
       if (error instanceof NotFoundError) {
-        if (this.returnMessages) return { [this.memoryKey]: [] };
-        return [];
+        return { [this.memoryKey]: [] };
```

Comment:
This doesn't look correct, if returnMessages is false it should instead maybe return `{ [this.memoryKey]: '' }` ?

Reply:
@nfcampos - we need to return an empty list not an empty string because memory is an array of messages.

Reply:
please let me know if you see an issue with this.

Reply:
No that's only true when returnMessages is true. When it is false memory returns a string (that concatenates the messages). See line 99 where it returns the result of calling getBufferString, ie. a string

Reply:
ah! thanks, updated to return this.returnMessages ? { [this.memoryKey]: [] } : { [this.memoryKey]: "" };